hierarc/Sampling/ParamManager/lens_param.py

```diff
@@ -16,6 +16,7 @@ def __init__(
         log_m2l_distribution="NONE",
         kappa_ext_sampling=False,
         kappa_ext_distribution="NONE",
```

Comment:
can't you define here an alternative distribution of kappa_ext instead of kappa_ext_alt?

Reply:
I could, but I think ultimately its the same.  I would still need to introduce the parameters that characterize a new alternative distribution, namely a mean and standard deviation.